stablehlo/dialect/StablehloOps.td

```diff
@@ -1460,7 +1460,8 @@ def StableHLO_BatchNormGradOp : StableHLO_Op<"batch_norm_grad", [NoSideEffect,
         "grad_offset"]>,
     AllShapesMatch<["operand", "grad_output"]>,
     AllElementTypesMatch<["operand", "grad_scale", "grad_offset"]>,
```

Comment:
Given that shape_inference.cc is checking for `SameElementTypeIgnoringFpPrecision`, do we need `AllElementTypesMatch` at all? Since ODS enforces all participating values to be tensors of `HLO_Float`, `SameElementTypeIgnoringFpPrecision` is achieved automatically, right? (Here and below).

Reply:
Good point!